src/makeRobot.js

```diff
@@ -39,6 +39,62 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name
+      }, chip version: ${this.version
+      }, wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: `
+      + `x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+    goForward(a = 1) {
+      if (a < 0) {
+        return this;
+      }
+      this.coords.y += a;
```

Comment:
```suggestion
      if (a > 0) {
        this.coords.y += a;
      }
```
Легким движением руки избавляемся от дублирования кода